llms/mlx_lm/evaluate.py

```diff
@@ -346,11 +361,8 @@ def main():
     )
     parser.add_argument(
         "--apply-chat-template",
```

Comment:
It was impossible to disable this before, so I've changed it to be off by default (which mirrors the `lm_eval` behavior)

Reply:
I'm not sure about defaulting it to off for instruct models.. it seems like you would always want this on for most models that are used regularly? Does it make sense to change this to `--ignore-chat-template` instead to be able to shut it off if needed?